instrumentation/opentelemetry-instrumentation-boto/src/opentelemetry/instrumentation/boto/__init__.py

```diff
@@ -136,27 +136,31 @@ def _common_request(  # pylint: disable=too-many-locals
                     attributes={"endpoint": endpoint_name}
                 )
 
+            # Original func returns a boto.connection.HTTPResponse object
+            result = original_func(*args, **kwargs)
+
             add_span_arg_tags(
                 span, endpoint_name, args, args_name, traced_args,
             )
 
-            # Obtaining region name
-            region_name = _get_instance_region_name(instance)
+            if span.is_recording():
```

Comment:
This check could be moved above add_span_arg_tags, since it looks like that method has a check for is_recording anyways.